experimental/veggieseasons/lib/styles.dart

```diff
@@ -3,171 +3,115 @@
 // found in the LICENSE file.
 
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
 import 'package:veggieseasons/data/veggie.dart';
 
 abstract class Styles {
-  static TextStyle headlineText(CupertinoThemeData themeData) => TextStyle(
-        color: themeData.textTheme.textStyle.color,
+  static CupertinoThemeData veggieThemeData = CupertinoThemeData(
+    textTheme: CupertinoTextThemeData(
+      textStyle: TextStyle(
+        color: CupertinoColors.label,
+        fontSize: 16,
+        fontWeight: FontWeight.normal,
+        fontStyle: FontStyle.normal,
         fontFamily: 'NotoSans',
+        letterSpacing: -0.41,
+        decoration: TextDecoration.none,
+      ),
+    ),
+  );
+
+  static TextStyle headlineText(CupertinoThemeData themeData) =>
+      themeData.textTheme.textStyle.copyWith(
         fontSize: 32,
-        fontStyle: FontStyle.normal,
         fontWeight: FontWeight.bold,
       );
 
-  static const minorText = TextStyle(
-    color: Color.fromRGBO(128, 128, 128, 1),
-    fontFamily: 'NotoSans',
-    fontSize: 16,
-    fontStyle: FontStyle.normal,
-    fontWeight: FontWeight.normal,
-  );
+  static TextStyle minorText(CupertinoThemeData themeData) =>
+      themeData.textTheme.textStyle.copyWith(
+        color: Color.fromRGBO(128, 128, 128, 1),
```

Comment:
It seems like we're using a regular color. Is "minorText" supposed to use the same color when you switch from light mode to dark mode? Same with "cardTitleText" and "cardCategoryText", etc,.

Reply:
`minorText` is used for the date in the Home Screen. It looks okay with both themes since it's grey.

The cards are all white with black text, which also works fine since they have a picture (and shadows).
I haven't changed any of these values. I assume it was intentional.

Reply:
I see. The cards all have very bright backgrounds because of the images they used. Make sense to opt-out of dark mode on the cards.